File: js/day32.js
function currentTime(){
  const time = new Date();
  return time.toLocaleTimeString();
}

function currentDate(){
  const date = new Date();
  return date.toLocaleDateString();
}

const time = currentTime();
const date = currentDate()
console.log(`Current date and time: ${date} ${time}`);

File: python/day32.py
from datetime import date
import time

def currentTime():
  t = time.localtime()
  formatted_time = time.strftime("%I:%M %p", t)
  return formatted_time

def currentDate():
  d = date.today()
  formatted_date = d.strftime("%d/%m/%Y")
  return formatted_date

print(f"Current date and time: {currentDate()} {currentTime()}")